### DISCUSSION TITLE
Err_Information_Leak

### DISCUSSION THREAD

**Discussion by JEA1310** [ACINQ/eclair#2084] [Q&A] [answered]
This is the error that keeps appearing on my closed channel when trying to recover funds. Please help me I'm recovering these funds from the channel (I recently moved my data to another phone which probably caused the issue).

**Comment by t-bast** [comment, marked as answer]
I guess you're not using eclair (server implementation) but are rather talking about eclair-mobile?
If so please open an issue on the eclair mobile repo: https://github.com/ACINQ/eclair-mobile

**Reply by JEA1310** [reply]
Hello. I went and asked there, but no one has answered me yet it has been 4 days to the post.
